### ISSUE TITLE
`sk`: Incorrect locale ordinal number settings

### ISSUE THREAD

**Issue by tukusejssirs** [romcal/romcal#186]
I have just noticed that the TS branch (maybe other branches too, but I did not check those) that the ordinal numbers are output as `[0-9]*º` (note the `º` character), but in Slovak (as well as Czech and mostly true about Hungarian), we use `.` (dot / full stop) instead.

Here is an example:

```json
// Current
"Streda 34º týždňa v Cezročnom období"  // wednesdayOfTheThirtyFourthWeekOfOrdinaryTime

// Correct
"Streda 34. týždňa v Cezročnom období"  // wednesdayOfTheThirtyFourthWeekOfOrdinaryTime
```

Another issue is that some of the l10n keys miss the `.` in `sk.ts` file (I did not notice this issue yet). I could fix the missing dots, but I think it should be fixed together with the former issue.

```json
# Current
sunday: '{{week}} adventná nedeľa'
octave: '{{count}} deň v oktáve narodenia Pána',
sunday: '{{count}} nedeľa po narodení Pána'
feria: '{{day}} {{week}} týždňa v Cezročnom období',
sunday: '{{week}} nedeľa v Cezročnom období'
sunday: '{{week}} pôstna nedeľa',
feria: '{{day}} po {{week}} veľkonočnej nedeli',
sunday: '{{week}} veľkonočná nedeľa',

# Correct
sunday: '{{week}}. adventná nedeľa'
octave: '{{count}}. deň v oktáve narodenia Pána',
sunday: '{{count}}. nedeľa po narodení Pána'
feria: '{{day}} {{week}}. týždňa v Cezročnom období',
sunday: '{{week}}. nedeľa v Cezročnom období'
sunday: '{{week}}. pôstna nedeľa',
feria: '{{day}} po {{week}}. veľkonočnej nedeli',
sunday: '{{week}}. veľkonočná nedeľa',
```

**Comment by pejulian** [comment]
Hey @tukusejssirs 

```
"Streda 34º týždňa v Cezročnom období"  // wednesdayOfTheThirtyFourthWeekOfOrdinaryTime
```

The output appears in this form because we have an underlying dependency to [dayjs](https://github.com/iamkun/dayjs) which defines ordinal number formatting for the `sk` locale [here](https://github.com/iamkun/dayjs/blob/dev/src/locale/sk.js#L27). This differs from our previously used dependency moment, which defines the ordinal function for the locale `sk` like [this](https://github.com/moment/moment/blob/develop/locale/sk.js#L146). Perhaps we need to open an issue in the dayjs repo to get this fixed.

> Another issue is that some of the l10n keys miss the . in sk.ts file (I did not notice this issue yet). I could fix the missing dots, but I think it should be fixed together with the former issue.

Ideally, we shouldn't need to add any prefixes (or suffixes) to the "week", "day" or "count" values in the locale file. This ordinal function [here](https://github.com/romcal/romcal/blob/_F-TypeScript/src/lib/Locales.ts#L263-L274) should take care of that by taking the given integer and formatting it to a locale compliant ordinal string to be inserted between the `{{` and `}}` braces in the locale file. In the case of `sk`, this formatted value should include the `.` character (incorrect now because of the issue described above).

**Comment by tukusejssirs** [comment]
> The output appears in this form because we have an underlying dependency to [dayjs](https://github.com/iamkun/dayjs) which defines ordinal number formatting for the `sk` locale [here](https://github.com/iamkun/dayjs/blob/dev/src/locale/sk.js#L27). This differs from our previously used dependency moment, which defines the ordinal function for the locale `sk` like [this](https://github.com/moment/moment/blob/develop/locale/sk.js#L146). Perhaps we need to open an issue in the dayjs repo to get this fixed.

Thanks! I have opened [iamkun/dayjs#831](https://github.com/iamkun/dayjs/issues/831).

> Ideally, we shouldn't need to add any prefixes (or suffixes) to the "week", "day" or "count" values in the locale file. This ordinal function [here](https://github.com/romcal/romcal/blob/_F-TypeScript/src/lib/Locales.ts#L263-L274) should take care of that by taking the given integer and formatting it to a locale compliant ordinal string to be inserted between the `{{` and `}}` braces in the locale file. In the case of `sk`, this formatted value should include the `.` character (incorrect now because of the issue described above).

Good to know! This information is quite important I think. Therefore the issue is the opposite: we need to remove the dots where they are still kept. I’ll create a PR against the TS branch.

**Comment by tukusejssirs** [comment]
Just a sidenote: It might happen that in one locale, the number might need to be ordinal, in other it might require cardinal number. We might need to take care of this (probable) issue.

**Comment by tukusejssirs** [comment]
This issue should be fixed by [`dayjs` v1.8.23](https://github.com/iamkun/dayjs/releases/tag/v1.8.23). I think 5hat can close this issue if we make that version the minimum version in romcal dependencies.

**Comment by tukusejssirs** [comment]
Closing, as the `dayjs` minimum version in romcal’s [`package.json` (L80)](https://github.com/romcal/romcal/blob/7c285beab3a2de46a4cf5bdaca67564132ee5d7c/package.json#L80) is already higher than `v1.8.23`.
